src/components/Button/Button.tsx

```diff
@@ -31,6 +31,7 @@ export type ButtonProps = CommonButtonProps & {
   title?: string;
   isCurrent?: boolean;
   currentStyles?: ButtonCurrentStyles;
+  role?: string;
```

Comment:
Confused by this - is this for accessibility? I can't tell where it's applied.

Reply:
Was a recommendation from @rightsaidjames for accessibility on selected buttons
![Screenshot 2023-09-20 at 11 37 19](https://github.com/oaknational/Oak-Web-Application/assets/91190841/b5901a8e-d8eb-408e-9b4e-2b63f568cc23)